docs/walkthrough_train.md

```diff
@@ -3,15 +3,142 @@ title: Train models with eScriptorium
+
+You can also directly read Kraken's documentation to find out more about some of its behaviors.
+
+### Ground Truth
+
+Ground truth is another way to call the training data. Ground Truth is considered to be the most perfect transcription expected from a transcription model and according to the transcription guidelines defined by a project.  
+
+### Epoch
+
+In machine learning, a model iterates over the training dataset several times. Each iteration on the training set is called an epoch. At the end of each epoch, Kraken models assess their performance against a testing set and adjust their learning strategy. To assess the performance of a model, an accuracy rate is calculated.  
+
+### Fine tuning  
+
+Fine tuning is the act of re-training a model on a new dataset, as opposed to training a model "[from scratch](https://kraken.re/master/ketos.html#from-scratch)" (using only training data).  
+
+You can check [Kraken's documentation on Fine tuning](https://kraken.re/master/ketos.html#fine-tuning) to better understand its default behavior.
+
+### Overfitting
+
+Overfitting is a common problem in machine learning. It designates the fact that a model seems to perform very well when tested on the training and testing data while failing to generalized to other examples. In some ways, it is as if the model learns "by heart" the training examples and becomes incapable to using the knowledge to new lines of text.  
+
+You can find more information on Kraken's ways to handle overfitting [here](https://kraken.re/master/training.html#training).
+
+### Accuracy and Errors
+
+A **Transcription model** accuracy is calculated at character level. It corresponds to the number of correctly recognized characters over the total number of characters to recognize. If a model has an accuracy of 90%, it will correctly recognize 9 characters out of 10.  
+
+The errors are expressed in eScriptorium as the total number of deleted/inserted/substituted characters over the total number of characters recognized by the model. Such a ratio is the basis to compute the [CER](https://towardsdatascience.com/evaluating-ocr-output-quality-with-character-error-rate-cer-and-word-error-rate-wer-853175297510) of a model, also known as "Character Error Rate".  
+
+If a model makes 330 errors out of 3330 characters to recognize, its CER will be equal to $\frac{330}{3330} \times 100$, which is 9.90%.
+
+The CER is inversely proportional to the accuracy: if a model has a character accuracy of 90%, it also has a CER of 10%.
+
+Even though eScriptorium offers a simple accuracy rate for **Segmentation models**, such metric is generally computed at pixel level (via IoU for example), which is less meaningful. For this reason, a segmentation model with an accuracy of 60% might still be a rather good model. It is usually necessary to try.  
```

Comment:
What does IoU mean ? You also use it a few lines below, but it is not clear (or maybe it's just me)

Reply:
I added a link to the corresponding Wikipedia article